### PR TITLE
The one probe

### PR DESCRIPTION
**Pull request by agnor99**

### CODE REVIEW

File: src/main/java/software/bernie/techarium/integration/ModIntegrations.java [hunk 1]
```diff
@@ -27,6 +28,15 @@
...
             ThermalCultivationIntegration::new).registerSelf();
 
+    static {
+        System.out.println("preTheOneProbe");
+    }
+
+    public static final Integration.Wrapper<TheOneProbeIntegration> THE_ONE_PROBE = Integration.Wrapper.of("theoneprobe",
+            TheOneProbeIntegration::new).registerSelf();
+    static {
+        System.out.println("postTheOneProbe");
```
Comment (bernie-g) [inline src/main/java/software/bernie/techarium/integration/ModIntegrations.java:38]: Probably should use a Logger and also make the logging messages better

File: src/main/java/software/bernie/techarium/integration/ModIntegrations.java [hunk 2]
```diff
@@ -42,4 +52,9 @@
     public static LazyOptional<ThermalCultivationIntegration> getThermalCultivation() {
         return THERMAL_CULTIVATION.get();
     }
+
+    public static LazyOptional<TheOneProbeIntegration> getTheOneProbe() {
+        System.out.println("getTheOneProbe");
```
Comment (bernie-g) [inline src/main/java/software/bernie/techarium/integration/ModIntegrations.java:57]: Remove unnecessary logging

**Review by bernie-g — CHANGES_REQUESTED** [review]

File: src/main/java/software/bernie/techarium/integration/ModIntegrations.java (hunk 1 above)
Comment (agnor99) [reply src/main/java/software/bernie/techarium/integration/ModIntegrations.java:38]: They were for debugging, missed to remove them apperently gonna fix that in hwyla

**Comment by bernie-g** [comment]
Using HYWLA PR instead.

**Pull request closed without merging**